server/routes/confirmedOnReport/confirmedOnReportChangeReport.test.ts

```diff
@@ -38,8 +38,7 @@ describe('GET /prisoner-placed-on-report - edited adjudication', () => {
       .expect('Content-Type', /html/)
       .expect(res => {
         expect(res.text).toContain('John Smith’s report has been changed')
-        expect(res.text).toContain('You must follow local processes.')
```

Comment:
as per the ts file comment - we must be able to get rid of lots of mocking for this test now?